HW06-annotations/src/main/java/ru/otus/Main.java

```diff
@@ -0,0 +1,9 @@
+package ru.otus;
+
+import ru.otus.testRunner.TestRunner;
+
+public class Main {
+    public static void main(String[] args) {
+        TestRunner.runTests("ru.otus.test.Tests");
```

Comment:
```suggestion
        TestResults result = TestRunner.runTests("ru.otus.test.Tests");
        result.logTestResults();
```

Правильно, что сделали класс `TestResults` для результатов теста.
Только его лучше здесь вернуть и отдельно вывести.
Подумайте как бы вы тестировали ваше решение. Как проверить, что в `TestResults` правильные значения? При выводе результатов внутри `runTests` сделать это будет не просто.